server/graphql/v2/mutation/TransactionMutations.ts

```diff
@@ -120,6 +120,7 @@ const transactionMutations = {
         FromCollectiveId: orderToUpdate.FromCollectiveId,
         CollectiveId: orderToUpdate.CollectiveId,
         HostCollectiveId: toAccount.approvedAt ? toAccount.HostCollectiveId : null,
+        UserId: req.remoteUser.id,
```

Comment:
Adding the user who rejected the contribution here so that we can eventually start using that in the timeline.